apps/android/boot.js

```diff
@@ -258,7 +285,7 @@
   }, 2000));
   NRF.on("disconnect", () => {
     // disable HRM/activity monitoring ('act' message)
-    GB({t:"act",stp:0,hrm:0,int:0}); // just call the handler to save duplication
+    GB({t:"act",stp:0,hrm:0,int:0,rt:1}); // just call the handler to save duplication
```

Comment:
This one isn't needed I think? It just calls the same function in boot.js rather than sending anything to Gadgetbridge - it's basically just replicating Gadgetbridge telling Bangle.js to turn off realtime steps

Reply:
Ah - you're right.